src/m_misc.c

```diff
@@ -3241,6 +3241,7 @@ void M_LoadDefaults (void)
   //      if the following lines will get printed or not
 
   I_InitPrintf();
+  I_AtExit(I_ShutdownPrintf, true);
```

Comment:
Could we move this into `I_InitPrintf()`?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1197/commits/9a71d177a7c3590208620514ce6b554092533c8d